pom.xml

```diff
@@ -31,6 +31,24 @@
             <artifactId>spring-boot-autoconfigure</artifactId>
             <version>2.6.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>2.6.7</version>
```

Comment:
it's better to have a unified spring boot starter version. For example, you have:
```
<artifactId>spring-boot-starter-parent</artifactId>
        <version>2.6.0</version>
```
and
```
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-autoconfigure</artifactId>
            <version>2.6.5</version>
        </dependency>
```
Also, in the Spring Boot project it is not required to specify the starter version